Tests/Unit/dng/find_mutation.cc

```diff
@@ -216,7 +216,7 @@ BOOST_FIXTURE_TEST_CASE(test_calculate_mutation_expected, TrioWorkspace) {
     BOOST_CHECK_SMALL(0.116189 - mutation_stats.mu1p, 1e-6);
 
     BOOST_CHECK_EQUAL("GGxGG>GT", mutation_stats.dnt);
-    BOOST_CHECK_EQUAL("LB-NA12878:Solexa-135852", mutation_stats.dnl);
+    BOOST_CHECK_EQUAL("LB/NA12878:Solexa-135852", mutation_stats.dnl);
```

Comment:
Looks like running `dng-call` on a vcf still assumes that names are "sample:library" instead of "sample/library".

Can we fix that in this PR too?